CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to NetPanzer
+
+## Bug Reports and Feature Requests
+
+Anyone may [open an
+issue](https://github.com/netpanzer/netpanzer/issues).
+
+## Discussion
+
+If you're not sure if something is a bug, you can search, or post on, the
+[Discussions](https://github.com/netpanzer/netpanzer/issues).
+
+### Discord
+
+You may participate on our [Discord server](https://discord.gg/kCAB2CMswd).
+It's important to remember that any discussions about bugs or desired features
+in a chat room usually scroll away and become forgotten, so it's best to
+create a ticket or discussion on GitHub (mentioned above) if you really want a
+bug fix or new feature.
+
+## Coding Standards
+
+Our goal is to use the [Google C++ Style
+Guide](https://google.github.io/styleguide/cppguide.html).
+
+A `.clang-format` file is located in `src`.
+
+You can format code automatically by using
+[clang-format](https://clang.llvm.org/docs/ClangFormat.html) with the
+following arguments:
+
+    clang-format TBD
+
+## Website
+
+TBD
+
+## Translating
+
+TBD
```

Comment:
something to consider with UI rewrite. Maybe something like: https://github.com/etlegacy/etlegacy/pull/2417/files#diff-ce46adfa098294bb2a360c8d17f7a3ac1ac92fd220ce7d266c8eec6d01478ecbL3642

which uses GNU tools and Transifex: https://explore.transifex.com/etlegacy/etlegacy/